src/interfaces/correction-interfaces/correct-name-interface.ts

```diff
@@ -9,7 +9,27 @@ export interface CorrectNameOptionIF {
 }
 
 export interface NrCorrectionIF {
+  legalType: string,
   nrNumber: string,
   legalName: string,
+  expiry: string,
+  status: string
   applicant: NameRequestApplicantIF | {}
 }
+
+export interface NrResponseIf {
+  // eslint-disable-next-line camelcase
+  entity_type_cd: string, // Ignore lint error on property name served from api.
+  expirationDate: string,
+  state: string,
+  applicants: {
```

Comment:
I think applicants is an array...?

Reply:
It's oddly named, from namex it is an object but i agree.. misleading as there is only one Name Request applicant.

Reply:
Oh, I was thinking of names[].

Just leave applicants then.